### ISSUE TITLE
Improve drawing widgets; Add "join demo applet" link

### ISSUE THREAD

**Issue by binarybottle** [ChildMindInstitute/mindlogger-app-backend#480]
- Adds a link on the applet list to join the "MindLogger Demo" applet
 - The MindLogger Demo applet contains an activity with each of the supported widgets, together with descriptions. Currently points to a schema on a branch on my fork (https://github.com/stufreen/schema-standardization/tree/widget-demo-activity) but that should be updated once the schema is merged into the main schema repo.
 - Cleans up the code for the drawing widget and makes some minor improvements

![image](https://user-images.githubusercontent.com/14841718/58724828-968d4e00-83ab-11e9-9e49-9a9964680b8e.png)

![image](https://user-images.githubusercontent.com/14841718/58724850-a311a680-83ab-11e9-9af4-fe707103b778.png)
